site/content/introduction/01-Introduction.md

```diff
@@ -17,29 +17,6 @@ You can create a cluster in minutes with just one command – **`eksctl create c
 
 ## Usage
 
-To download the latest release, run:
```

Comment:
Something is wrong here - two simultaneous 'Introduction' sections have been created. 
<img width="824" alt="Screenshot 2019-06-17 at 16 36 51" src="https://user-images.githubusercontent.com/92439/59612952-4ee22280-911e-11e9-8542-6d469dfa65ee.png">

Reply:
This happens when there are two files in the introduction folder with the same title (second line in the file, not the name)

Reply:
I don't see it in the preview. Maybe you have two files locally?

Reply:
Oh, how weird, it seems to be a bug in the hugo server - if I refresh the page the second 'Introduction' section disappears, but if I click around for a while I can make it reappear..
Anyways, my apologies, this is not an issue.